### ISSUE TITLE
Bind for 0.0.0.0:80 failed: port is already allocated

### ISSUE THREAD

**Issue by vodish** [bitrixdock/bitrixdock#228]
Не запускается что-то...
 
 - Container web_server  Starting                                                     1.9s 
Error response from daemon: driver failed programming external connectivity on endpoint web_server (3e506b7472216f49c47051b740ff26a7e6e97402148dfc4a4d8e183ae1e50e83): Bind for 0.0.0.0:80 failed: port is already allocated

![image (1)](https://github.com/user-attachments/assets/186e3615-c490-4a36-904a-bb49b00f959d)

**Comment by Elnadrion** [comment]
Это значит что на хост машине порт 80 уже занят чем-то

**Comment by vodish** [comment]
Переделал порты. Вроде почти все запустилось.

![image](https://github.com/user-attachments/assets/71f1f59c-56c4-450a-bf77-be62499f1cf2)

А что дальше с этим делать?

![image](https://github.com/user-attachments/assets/30ed1aa1-715d-4417-aeae-c56dd240a5d7)

**Comment by Elnadrion** [comment]
> А что дальше с этим делать?

Прочитать документацию xD

**Comment by Elnadrion** [comment]
Да. Или можно в `.env` файле перемапить этот путь туда, где уже лежит развернутый сайт

**Comment by vodish** [comment]
Большое человеческое спасибо!

**Comment by churneya** [comment]
может стоит закрыть issue?
